components/AccountingCategorySelect.tsx

```diff
@@ -15,7 +15,7 @@ import { ACCOUNTING_CATEGORY_HOST_FIELDS } from './expenses/lib/accounting-categ
 import { Command, CommandEmpty, CommandGroup, CommandInput, CommandItem, CommandList } from './ui/Command';
 import { Popover, PopoverContent, PopoverTrigger } from './ui/Popover';
 
-type RequiredHostFields = Pick<Host, 'id' | 'slug'> & {
+type RequiredHostFields = Pick<Host, 'id' | 'slug' | 'type'> & {
```

Comment:
`isAdminOfCollective` requires the type to properly check if the account is a project or event (and consequently check the parent). I added the `type` for accounts in some queries to reflect that.